sass.md

```diff
@@ -80,6 +80,38 @@ body {
 }
 ```
 
+#### with default values
+
+```scss
+@mixin pad($n: 10px) {
+  padding: $n;
+}
+```
+
+```scss
+body {
+  @include pad(15px);
+}
+```
+
+#### with a default variable
+
+$default-padding: 10px;
```

Comment:
Can you add a code fence for scss around this?

I'd also like to add a comment in here to signify this is setting a default value:

```
// Set a default value
$default-padding: 10px;
```